src/components/DiscoveryPage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Container, Box, Typography, makeStyles } from '@material-ui/core';
+import { Navigation } from '.';
+import Assets from '../assets';
+
+const useStyles = makeStyles((theme) => ({
+    discoveryPageTitle: {
+        color: theme.palette.common.black,
```

Comment:
Setting `color` like this shouldn't be necessary for typography, as it is already the default.